src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+    <PropertyGroup>
+        <TargetFramework>$(LatestTargetFramework)</TargetFramework>
+        <!-- Workaround https://github.com/microsoft/MSBuildSdks/issues/469 -->
+        <TargetFrameworks />
```

Comment:
While you link to a workaround, I find unsetting the `TargetFrameworks` property the right to do anyway. It makes the build slightly faster as the outer build doesn't need to be evaluated and ran.